protostar/commands/test/cheatcodes/store_cheatcode.py

```diff
@@ -0,0 +1,80 @@
+from typing import Any, Callable, List, Optional
+from starkware.starknet.public.abi import get_storage_var_address
+from starkware.starknet.storage.starknet_storage import BusinessLogicStarknetStorage
+from starkware.crypto.signature.fast_pedersen_hash import pedersen_hash
+
+from protostar.commands.test.starkware.cheatcode import Cheatcode
+
+
+ADDR_BOUND = 2**251 - 256
+
+
+class StoreCheatcode(Cheatcode):
+    @property
+    def name(self) -> str:
+        return "store"
+
+    def build(self) -> Callable[..., Any]:
+        return self.store
+
+    def store(
+        self,
+        target_contract_address: int,
+        variable_name: str,
+        value: List[int],
+        key: Optional[List[int]] = None,
+    ):
+        key = key or []
+        variable_address = self.calc_address(variable_name, key)
+        if target_contract_address == self.contract_address:
+            for i, val in enumerate(value):
+                self.store_local(variable_address + i, val)
```

Comment:
Can you move the for loop to the `store_local` and add `store_remote` method?

Reply:
Yes we should